keycloak/keycloak-admin-ui
# Ream-settings-> User profile -> Create attribute

## Motivation
Create attribute as per designs https://marvelapp.com/prototype/5eed532/screen/82246809

## Brief Description
Most of the functionality is implemented, but due to this PR getting large I have opened a couple of issues for the outstanding work:

- Permission section - checkboxes don't return 'admin' and 'user' when selected - these values are expected by the BE and also additional logic is required for this design https://marvelapp.com/prototype/5eed532/screen/82246809 #2266

- Validations section - added validations not being detected as form values and cannot be sent to the BE #2267
- Validations section - deleting validations not implemented yet due to the above

- General Settings section - as per the old console, setting the "Required" toggle to false should hide the "Required For" and the "Required When" fields #2269

- Editing created attribute not implemented yet #2268

## Verification Steps
1. Go to realm-settings, general tab and select User Profile Enabled
2. Select the User Profile tab and click 'create attribute'
3. Verify that the page is as per designs, complete the form and click save. You should still be able to save the new attribute despite the outstanding work.

## Checklist:
